gravitee-apim-gateway/gravitee-apim-gateway-handlers/gravitee-apim-gateway-handlers-api/src/main/java/io/gravitee/gateway/handlers/api/services/SubscriptionService.java

```diff
@@ -70,6 +73,13 @@ public void save(Subscription subscription) {
             saveInCacheByApiAndClientId(subscription);
         }
         saveInCacheBySubscriptionId(subscription);
+
+        //TODO: If subscription's type is "subscription", forward to the subscription dispatcher
```

Comment:
No it doesn't, I was looking at finding a good way to know the type of subscription without necessarely adding a new type field, but it sounds I'll not be able to make it.
For going ahead, I relied on the configuration, because that was the easiest to discover a `subscription` type of subscription right now.